tests/algorithms/test_algorithm_resumption.py

```diff
@@ -15,7 +15,7 @@
 
 
 @pytest.mark.timeout(180)
-@device('cpu')
+@device('gpu')
```

Comment:
#941 did have CPU tests, but disabled them for all but the most simple of models. Running the CPU tests here take:
```
20 passed, 8 skipped, 18 xfailed, 52 warnings in 67.71s (0:01:07)
```
on an 8-core machine, and this will only grow over time. I'll keep them on GPU since that is the main use case.